test/format/snap.c

```diff
@@ -178,6 +178,11 @@ snap_verify(WT_CURSOR *cursor, TINFO *tinfo, SNAP_OPS *snap)
 	}
 
 	/* Things went pear-shaped. */
+#ifdef HAVE_DIAGNOSTIC
+	fprintf(stderr,
+	    "snapshot-isolation error: Dumping tree to %s\n", g.home_treedump);
+	(void)__wt_debug_tree_cursor(cursor, g.home_treedump);
+#endif
```

Comment:
This is going to hit some of the "function return value unchecked" complaints, should have `testutil_check()`.